src/input-select/types.ts

```diff
@@ -60,6 +60,8 @@ export interface InputSelectProps<T, V>
     displayValueExtractor?: (option: T) => string | undefined;
     /** Function to convert value into a string */
     valueToStringFunction?: (value: V) => string | undefined;
+    /** Function to render selected custom component */
+    renderCustomSelectedItem?: (item: string | V) => JSX.Element | undefined;
```

Comment:
per above comment, amend typing

also rename to be consistent with the other props 

```suggestion
    renderCustomSelectedOption?: (item: T) => JSX.Element | undefined;
```